apiary.apib

```diff
@@ -536,6 +536,17 @@ Get details for a single user.
             + function: `Sales` (string)
             + time_zone: `Europe/Brussels` (string)
 
+### users.invite [GET /users.invite]
+
+Invite a single user to your account
+
++ Request (application/json)
+
+    + Attributes (object)
+        + email: `john@teamleader.eu` (string, required)
```

Comment:
Should this also contain an `account_id`?

Reply:
Any reason? The user that invites a new user is already authenticated so, we know for which account this user is invited. 😉